datalake/backend/endpoints/parameter_manager.py

```diff
@@ -16,6 +17,9 @@ class ResourceName(Enum):
 
 
 def get_param(parameter: ResourceName):
-    path = f"/{ENV}/{parameter.value}"
-    ssm_parameter = ssm_client.get_parameter(Name=path)
-    return ssm_parameter["Parameter"]["Value"]
+    path = f"/{ENV}/{parameter.name}"
+
+    try:
+        return ssm_client.get_parameter(Name=path)["Parameter"]["Value"]
+    except ClientError as e:
+        print(e.response["Error"]["Code"])
```

Comment:
I don't think we want to ignore this exception. If any kind of error (permission, no such parameter, etc) is going to result in a `ClientError` then we should simply not catch it and let the application crash, because it would be an implementation error. [My original suggestion](https://github.com/linz/geospatial-data-lake/pull/198#discussion_r565647975) was made under the assumption that `get_parameter` might not throw an exception in certain error cases such as the parameter not existing. If that is not the case we can simply remove the error handling.